cmd/utils.go

```diff
@@ -965,3 +966,18 @@ func auditLogInternal(ctx context.Context, bucket, object string, opts AuditLogO
 	ctx = logger.SetAuditEntry(ctx, &entry)
 	logger.AuditLog(ctx, nil, nil, nil)
 }
+
+// hashStr2Int - Returns an integer hash of the given string
+func hashStr2Int(s string) uint32 {
+	h := fnv.New32a()
```

Comment:
Use crc32 (IEEE) if you just need a fast hash. An order of magnitude faster.

Just remove the entire func and use `crc32.ChecksumIEEE([]byte(s))` at the call site instead of having this func.

Reply:
Thanks, will do.